terraform/modules/rds_migration/ecs.tf

```diff
@@ -0,0 +1,128 @@
+#tfsec:ignore:aws-ecs-enable-container-insight:TODO CLDC-2542 enable container insights if necessary for logging/monitoring
+resource "aws_ecs_cluster" "this" {
+  #checkov:skip=CKV_AWS_65:TODO CLDC-2542 enable container insights if necessary for logging/monitoring
+  name = "${var.prefix}-db-migration"
+}
+
+resource "aws_ecs_task_definition" "ad_hoc_db_migration" {
+  #checkov:skip=CKV_AWS_336:using readonlyRootFilesystem to true breaks the app, as it needs to write to app/tmp/pids for example
+  family                   = "${var.prefix}-ad-hoc-db-migration"
+  cpu                      = var.db_migration_task_cpu
+  execution_role_arn       = var.ecs_task_execution_role_arn
+  memory                   = var.db_migration_task_memory #MiB
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  task_role_arn            = var.ecs_task_role_arn
+
+  container_definitions = jsonencode([
+    {
+      name      = "db-migration"
+      essential = true
+      image     = var.ecr_repository_url

```

Comment:
Currently this won't return a valid image AFAIK. For the regular ECS app task this gets updated by the deployment pipeline I believe.